### DISCUSSION TITLE
空格可以像其他按键一样添加功能吗

### DISCUSSION THREAD

**Discussion by Attiv** [imfuxiao/Hamster#71] [Q&A]
比如上划次选，这个还是挺实用的

**Comment by imfuxiao** [comment]
对一个程序员来说, 没有不可能的事情....哈哈...
等先忙完这段时间了, 我看看怎么搞定它.

**Reply by Attiv** [reply]
这个不急的话我有空了看看吧😂，最近也没啥时间看

**Reply by imfuxiao** [reply]
嗯嗯. 不急

**Reply by qiyueyouni** [reply]
可不可以加入功能模块呢比如复制，黏贴，剪贴，全选等呀

**Reply by imfuxiao** [reply]
这个有考虑过, 但是涉及隐私, 需要慎重思考, 是否真的需要加这个功能.

**Reply by qiyueyouni** [reply]
> 这个有考虑过, 但是涉及隐私, 需要慎重思考, 是否真的需要加这个功能.

这个可以自定义选择。看用户自己需求。这个感觉是最适宜的功能。对输入法太重要了。比如下滑那个按键黏贴

**Reply by qiyueyouni** [reply]
> 这个有考虑过, 但是涉及隐私, 需要慎重思考, 是否真的需要加这个功能.

现在那个输入法没有黏贴版呢？而且只是需要能自定义功能呢？要不然太麻烦了

**Reply by imfuxiao** [reply]
嗯嗯, 我先加入到需求列表中.

**Reply by qiyueyouni** [reply]
> 嗯嗯, 我先加入到需求列表中.

好的。非常迫切需要。这样我就能替代现在的输入法了

**Reply by qiyueyouni** [reply]
就是上下滑动数字符合里面的功能建

**Reply by imfuxiao** [reply]
已关联 #92
